sdk/keyvault/Azure.Security.KeyVault.Keys/src/EcKeyCreateOptions.cs

```diff
@@ -26,6 +26,11 @@ public EcKeyCreateOptions(string name, bool hsm)
             {
                 KeyType = KeyType.EllipticCurve;
             }
+
+            if (curveName != default)
+            {
+                Curve = curveName;
+            }
```

Comment:
What's the reason for this check?  When the key is actually created this is nullable should the parameter curveName and the property Curve be as well?

Reply:
Sanity for me, but nothing major. I can delete it